test/unittests/test_excel_to_properties.py

```diff
@@ -0,0 +1,63 @@
+"""unit tests for excel to properties"""
+import os
+import unittest
+
+from openpyxl import Workbook
+
+from knora.dsplib.utils import excel_to_json_properties as e2j
+
+
+class TestExcelToProperties(unittest.TestCase):
+
+    def setUp(self) -> None:
+        """Is executed before each test"""
+        os.makedirs('testdata/tmp', exist_ok=True)
+
+    def test_excel2json(self) -> None:
+        excelfile = "testdata/Properties.xlsx"
+        outfile = "testdata/tmp/_out_properties.json"
+        e2j.properties_excel2json(excelfile, outfile)
+        self.assertTrue(os.path.exists(outfile))
+
+    def test_row_to_prop(self) -> None:
+        wb = Workbook()
+        ws = wb.create_sheet("Tabelle1")
+        row = (
+            "hasAnthroponym  ",
+            " hasValue, dcterms:creator   ",
+            " TextValue ",
+            "anthroponym",
+            "",
+            "Anthroponyme",
+            "",
+            " A strange chance put me in possession of this journal.  ",
+            "",
+            "",
```

Comment:
adding a number only/umlaut only field here would probably get lost because of the regex only looking as a-zA-Z

Reply:
No, it can't get lost, because as long as at least one cell contains at least one letter, the entire row is processed.

Reply:
It would be best to have one or several tests with values with these characters as well.